Formula/zsh.rb

```diff
@@ -46,6 +46,7 @@ def install
       --enable-multibyte
       --enable-zsh-secure-free
       --with-tcsetpgrp
+      DL_EXT=bundle
```

Comment:
Otherwise those `.so` files seem weird on macOS, IMO.

Actually, MacPorts is doing exactly the same.

https://github.com/macports/macports-ports/blob/master/shells/zsh/Portfile#L57